src/util/auth/google_authentication_repository.ts

```diff
@@ -0,0 +1,106 @@
+import { secureStore } from "@storage";
+import { get } from "svelte/store";
+
+const storedAccessToken = secureStore<string>("access_token", null);
+const storedRefreshToken = secureStore<string>("refresh_token", null);
+
+// Scopes must match the scopes configured in the Google Developers Console.
+const SCOPES = [
+  "https://www.googleapis.com/auth/photoslibrary.readonly",
+].join("_");
+
+interface AuthenticationResponse {
+  accessToken: string,
+  refreshToken: string
+};
+
+const save = (accessToken: string, refreshToken: string) => {
+  console.log(`Saving access_token:(${accessToken}), refresh_token:(${refreshToken})`);
+  storedAccessToken.set(accessToken);
+  storedRefreshToken.set(refreshToken);
+};
+
+const authenticate = async (authorizationCode: string): AuthenticationResponse => {
```

Comment:
```suggestion
const authenticate = async (authorizationCode: string): Promise<AuthenticationResponse> => {
```
This fixes a typeError :)